skbio/stats/composition.py

```diff
@@ -1105,6 +1236,76 @@ def _gram_schmidt_basis(n):
     return basis.T
 
 
+def sbp_basis(sbp):
+    """
+    Builds an orthogonal basis from a sequential binary partition (SBP). As
+    explained in [1]_, the "SBP describes the D−1 orthogonal (geometrically
+    independent) balances between parts and groups of parts. The SBP is a
+    :math:`(D - 1) \times D` matrix, in which parts labeled '+1' (group
+    numerator) are balanced with parts labeled '−1' (group denominator). A part labeled '0' is excluded from the balance between parts. The composition is
```

Comment:
This line is quite large and will probably throw a pep8 error.  Could we have a new line?

Reply:
I have pushed the version written here (with a minor modification). https://twitter.com/jamietmorton/status/930288289419902976